### DISCUSSION TITLE
Sample post

### DISCUSSION THREAD

**Discussion by chiche-ds** [chiche-ds/30-Day-Terraform-challenge-#3] [Learning resources]
Hi everyone,

I came across this great article on Infrastructure as Code and thought it might be useful for those of you who want to dive deeper into the concept: [Link to Article]

Additionally, here are a few other resources that I found helpful:
- **[Terraform: Up & Running](https://www.amazon.com/Terraform-Running-Infrastructure-Configuration-Management/dp/1492046906)** - A comprehensive book on Terraform.
- **[HashiCorp Learn](https://learn.hashicorp.com/terraform)** - Official Terraform tutorials and guides.

Feel free to share any other resources you’ve found helpful!

Best,
@chiche-ds

**Comment by Assan-zidan** [comment]
Hi @chiche-ds 

Good 👍
